frontend/src/metabase/entities/collections/collections.ts

```diff
@@ -102,7 +106,7 @@ const Collections = createEntity({
       Collections.actions.update(
         { id },
         { archived },
-        undo(opts, "collection", archived ? "archived" : "unarchived"),
+        undo(opts, "collection", archived ? "trashed" : "restored"),
```

Comment:
localize the strings? Not sure if that is needed here

Reply:
resolved using the function i inlined a few comments before this one